### ISSUE TITLE
New display style: Human Case

### ISSUE THREAD

**Issue by hellow554** [frozenlib/parse-display#18]
I would like to see a "Human Case", where instead of using a `_` or `-` as seperator I'd like to have a ` ` (space, 0x20, ...).

Example:

```rust
#[derive(Display, FromStr, PartialEq, Debug)]
enum StyleExample {
  #[display(style = "Human Case")]
  VarH,
}

assert_eq!(StyleExample::VarH.to_string(), "Var H");
```

**Comment by frozenlib** [comment]
Thanks for posting the issue.

I would like to avoid using terms that are not common.
Could you please tell me the origin of the term "Human Case", if any?

If there is no origin, then [heck crate](https://crates.io/crates/heck) uses the term "Title Case", and I would like to match it.

**Comment by hellow554** [comment]
>  Could you please tell me the origin of the term "Human Case", if any?

There isn't any. I just searched for something that could describe it ;)

If "Tile Case" is the "official" term I don't mind using it.

**Comment by frozenlib** [comment]
For now, I've made ‘[#display(style = "Title Case")]‘ available with 8cfd5b0a78ec60f5a879e22a9b9526a738ccd5a8 & 7a178be6397820847db533873074d6846d849f3b.

If you find a more appropriate style name, please let me know.

**Comment by hellow554** [comment]
Wow, amazing! This issue can be closed. 

Thanks a lot for the quick and easy work! :heart:

**Comment by hellow554** [comment]
One thing:

https://github.com/frozenlib/parse-display/blob/7a178be6397820847db533873074d6846d849f3b/parse-display-derive/src/lib.rs#L807-L817

Would you like to add `TITLE CASE`, `Title case` and `title case` as well?

**Comment by frozenlib** [comment]
Thanks.

It certainly looks like I should add it.
I've fixed it now in 4893a8a536714d1e8440be47d727990bc9978a20.

**Comment by frozenlib** [comment]
Oops.

`title case` is now `title Case`, so I'll fix that right away.
